examples/demo_grammars.ipynb

```diff
@@ -0,0 +1,1010 @@
+{
```

Comment:
This is fantastic! The notebook looks impressive.
- I can see that graphviz or pretty-printing would really help. I'll prioritize working on that.
- I noticed that you ran HyperoptCV on the first three examples, which is really nice. Would it also be possible to run it on the fourth example?
- What is that ERROR about SelectKBest? Maybe comment out the g.supervised line?
- Looks like we need SkImputer and MissingIndicator to fully replicate the prior art. What would it take to add them?
- Typo: "exercice" -> "exercises"